pkg/mount/gpt/gpt_test.go

```diff
@@ -48,6 +48,8 @@ func InstallGPT() {
 	}
 }
 
+// GPT is GUID Partition Table, so technically, this test name is
+// Test Guid Partition Table Table. :-)
```

Comment:
We could also name it TestGPT but I'm also fine with the comment

Reply:
It's ok, I think that change is out of scope, I just thought it was funny.